### PR TITLE
Adapt createAsset to composite extrinsics and batching

### PR DESCRIPTION
**Pull request by monitz87**
### Description

- Use the new `create_asset_with_custom_type` extrinsic
- Adapt the `createAsset` procedure to use batching

My bad with the wrong commit message, I thought I had already committed the logic

### Checklist

- [ ] Updated the Readme.md (if required) ?

### CODE REVIEW

File: src/api/procedures/createAsset.ts [hunk 1]
```diff
@@ -169,60 +174,142 @@ export async function prepareCreateAsset(
...
     fee = new BigNumber(0);
   } else if (status === TickerReservationStatus.Free) {
-    const [registerTickerFee, createAssetFee] = await Promise.all([
-      context.getProtocolFees({ tag: TxTags.asset.RegisterTicker }),
-      context.getProtocolFees({ tag: TxTags.asset.CreateAsset }),
-    ]);
-    fee = registerTickerFee.plus(createAssetFee);
+    fee = await addManualFees(
+      new BigNumber(0),
+      [TxTags.asset.RegisterTicker, TxTags.asset.CreateAsset],
```
Comment (VictorVicente) [inline src/api/procedures/createAsset.ts:194]: Why is it adding `RegisterTicker` fee? What does it happen if ticker was registered previously?

**Review by prashantasdeveloper — APPROVED** [review]
LGTM

**Review by VictorVicente — APPROVED** [review]

**Pull request merged**